README.md

```diff
@@ -1,3 +1,5 @@
+*Important* : EosJs2 is under active development and should be considered beta, improvements and enhancements may break or remove existing functionality, be sure to lock your dependencies.
+
```

Comment:
Small edits:

*Important*: `eosjs2` is under active development and should be considered beta. Improvements and enhancements may break or remove existing functionality. Be sure to lock your dependencies.

I don't think we've used `EosJs2` before. Sticking with `eosjs2`. Wrapping the name in backticks optional. I have no strong feelings there.